README.rst

```diff
@@ -523,8 +525,6 @@ Choose from the list of available configurations:
 The ``--dry-run`` option displays the files that need to be
 fixed but without actually modifying them:
 
-.. code-block:: bash
```

Comment:
When this PR will be ready to merge I will ask you for squash commits to be one commit per author